google-cloud-tools-plugin/src/com/google/gct/idea/appengine/validation/ApiNamespaceInspection.java

```diff
@@ -75,7 +75,12 @@ public void visitAnnotation(PsiAnnotation annotation) {
           return;
         }
 
-        if(!annotation.getQualifiedName().equals(GctConstants.APP_ENGINE_ANNOTATION_API_NAMESPACE)) {
+        String annotationQualifiedName = annotation.getQualifiedName();
+        if (annotationQualifiedName == null) {
```

Comment:
Optional: if you switch the order of the comparison so the constant comes first, then you don't need the null check. I.e. 
 if (!GctConstants.APP_ENGINE_ANNOTATION_API_NAMESPACE.equals(annotation.getQualifiedName()))